wgked

```diff
@@ -3,27 +3,14 @@ from flask import Flask, render_template, jsonify, request
 import base64
 import json
 import re
+import yaml
+import argparse
 
-app = Flask(__name__)
 
-WG_PUBKEY_PATTERN = re.compile("^[A-Za-z0-9+/]{42}[A|E|I|M|Q|U|Y|c|g|k|o|s|w|4|8|0]{1}=$")
-
-SEGMENTS=[
-'ffmuc_freising',
-'ffmuc_gauting',
-'ffmuc_muc_cty',
-'ffmuc_muc_nord',
-'ffmuc_muc_ost',
-'ffmuc_muc_sued',
-'ffmuc_muc_west',
-'ffmuc_uml_nord',
-'ffmuc_uml_ost',
-'ffmuc_uml_sued',
-'ffmuc_uml_west',
-'ffmuc_welt'
-]
+app = Flask(__name__)
 
 def is_valid_wg_pubkey(pubkey):
+  WG_PUBKEY_PATTERN = re.compile("^[A-Za-z0-9+/]{42}[A|E|I|M|Q|U|Y|c|g|k|o|s|w|4|8|0]{1}=$")
```

Comment:
This makes compiling the regex useless as it will not be reused between validation calls.

Reply:
@lqb

Reply:
Fixed in 5e453f483a6ac30d78b2578b7cea55435eb01a56